pkg/util/informermanager/informermanager.go

```diff
@@ -134,11 +135,15 @@ func (m *informerManager) worker(ctx context.Context) {
 		return
 	}
 
-	err, needReenqueue := m.processFTC(ctx, ftc)
+	err, needReenqueue, delay := m.processFTC(ctx, ftc)
 	if err != nil {
 		if needReenqueue {
 			logger.Error(err, "Failed to process FederatedTypeConfig, will retry")
-			m.queue.AddRateLimited(key)
+			if delay > 0 {
+				m.queue.AddAfter(key, delay)
+			} else {
+				m.queue.AddRateLimited(key)
+			}
```

Comment:
Should we ignore delay if err != nil? If we don't, do want to reset backoff at line 143?